packages/blockchain-wallet-v4-frontend/src/modals/AddBitcoinWallet/template.js

```diff
@@ -22,7 +22,9 @@ const AddBitcoinWallet = (props) => {
   const { position, close, submitting, invalid, wallets, ...rest } = props
   const { onSubmit } = rest
 
-  const unique = (props) => map(prop('label'), wallets).indexOf(props) > -1 ? 'Wallet name is already taken.' : undefined
+  const unique = (value) => {
```

Comment:
what does this fix?

Reply:
this file should not have been committed. i will uncommit.